src/main/kotlin/lotto/domain/LottoMachine.kt

```diff
@@ -1,26 +1,23 @@
 package lotto.domain
 
 class LottoMachine {
-    fun buy(money: Money): List<Lotto> {
-
-        val ticketCount: Int = money.price / TICKET_PRICE
-        val tickets = mutableListOf<Lotto>()
-
-        repeat(ticketCount) {
-            tickets.add(issueTicket())
-        }
-        return tickets
+    fun buy(money: Money, manualTickets: Tickets?): Tickets {
+        val autoTicketCount = canBuyTicketCount(money) - (manualTickets?.count() ?: 0)
+        val autoTickets = Tickets(List(autoTicketCount) { issueTicket() })
+        return manualTickets?.merge(autoTickets) ?: autoTickets
```

Comment:
아래와 같은 코드로 바꿔볼 수 있을까요?
```suggestion
        return manualTickets?.let { it + autoTickets } ?: autoTickets
```
그런데 manualTickets 가 not null 이라면 (empty collection 이라면) 좀 더 간결하게 표현 가능할 것 같습니다.

```suggestion
        return manualTickets + autoTickets
```

InputView 에서 다음과 같은 피드백을 남겼는데 관련해서 고민해 보면 좋겠습니다.
>  null 을 사용하지 않고 0개의 `Tickets` 를 사용할것이냐 vs null 을 사용하고 1개 이상의 `Tickets` 를 보장할것이냐